index.html

```diff
@@ -70,8 +70,7 @@
 	<h1 class="mono">sepheo coop</h1>
 	<div id="about">
 		<p>SEPHEO (Self-Employed People Helping Each Other) Coop is a mutual society for self-employed
-			professionals. We make connections between coops and freelancers who want to work
-			in the coop economy.
+			professionals. We make it easy for freelancers to connect with coops and other aligned organisations.
```

Comment:
Happy with this, although personally I don't think alignment is easy to check nor is it strictly required - we should just encourage it.

Reply:
agreed